client/src/main/java/io/pravega/client/stream/impl/PravegaDefaultCredentials.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.stream.impl;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.jasypt.util.password.StrongPasswordEncryptor;
+
+public class PravegaDefaultCredentials implements PravegaCredentials {
+    private final String password;
+    private final String userName;
+
+    public PravegaDefaultCredentials(String password, String userName) {
+        StrongPasswordEncryptor passwordEncryptor = new StrongPasswordEncryptor();
+        this.password = passwordEncryptor.encryptPassword(password);
```

Comment:
This is going to be using a random salt. So the server will have no way of comparing this to anything. 
We need to either have the server first tell the client the salt, or have a way to derive it